src/bci_build/package/python/README.md.j2

```diff
@@ -0,0 +1,57 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Python](https://www.python.org/) is an interpreted, interactive, object-oriented, open-source programming language. It incorporates modules, exceptions, dynamic typing, high-level dynamic data types, and classes. It provides interfaces to many system calls, libraries, and various window systems, and it is extensible in C or C++. It is also usable as an extension language for applications that require programmable interfaces.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
```

Comment:
to me "In order to deploy.." reads a bit nicer.

Reply:
@dmpop WDYT?

Reply:
maybe

```
Deploying an application by installing dependencies, copying the sources and configuring the entrypoint:
```

would be another way to go imho.

Reply:
_To X, do Y_ is a standard neutral style. It's not about making things sound nicer or piling gerunds. It's making the text simple to read and instructions easy to follow.

Also, keep in mind that we have a style checker that ensures that we stick to the style defined in our style guide. It will flag your suggestions as an anomaly.

Reply:
I see, thanks for the explanation. Indeed, "vale" is flagging it as too wordy. alright. The suggestion that vale comes up with reads well for me as well: 

```
Deploying a python application involves installing dependencies, copying the source files, and configuring the entry point:
```